### DISCUSSION TITLE
ory_kratos_continuity cookie has no domain set

### DISCUSSION THREAD

**Discussion by akkie** [ory/kratos#2560] [Q&A] [answered]
Hi,

why has the ory_kratos_continuity cookie no domain set? Is there a special reason?

I run Kratos under a subdomain and because of the missing domain, the ory_kratos_continuity cookie will be set under the domain which serves the UI. After the redirect back from the OIDC provider, I get the error: no resumable session found

If I manipulate the cookie before it get's set in the browser and set the domain, all works as expected.

All other cookies also work as expected, based on the configuration in the config.

Best regards,
Christian

**Comment by aeneasr** [comment, marked as answer]
Hi Christian, sorry for the late reply - we could use the global cookie domain for the continuity cookie as well! Usually it doesn't need special configuration but there are probably edge cases where this can happen.

**Reply by akkie** [reply]
@aeneasr Thanks for your answer. Should I create an issue?

**Reply by aeneasr** [reply]
Issue or PR, whatever you prefer :)

**Reply by edelmann7** [reply]
@akkie did you by chance create the PR or issue?

**Reply by vinckr** [reply]
Hey @edelmann7 
I believe this is the issue: https://github.com/ory/kratos/issues/2713
